doc/rst/platforms/macosx.rst

```diff
@@ -4,40 +4,57 @@
 Using Chapel on Mac OS X
 ========================
 
-By and large, Chapel can be used on a Macintosh running OS X just like
-any other platform -- refer to the :ref:`chapelhome-quickstart` for more
-information on building and using Chapel.
+There are two main approaches for using Chapel on Mac OS X:
+
+1) Install via Homebrew_.  For existing Homebrew users, this is the
```

Comment:
drop "existing"?

Reply:
"current Homebrew users"?  My thought was that if you don't already use homebrew, getting started with homebrew and then using it to install Chapel may be a similar amount of work to installing from source with an inferior result...

Reply:
Yeah maybe. I was just saying that existing/current etc doesn't really add much there. But no big deal.

Reply:
Oh, now I see.  Makes sense, thanks.